### ISSUE TITLE
Remove language if we have Redirect to false in config

### ISSUE THREAD

**Issue by xDveGax** [wiziple/gatsby-plugin-intl#106]
Hi, i have my options like:  
`resolve: `gatsby-plugin-intl`,
options: {
        path: `${__dirname}/src/intl`,
        languages: [`es`, `en`],
        defaultLanguage: `en`,
        redirect: false,
 },`

Can we do in this line:
https://github.com/wiziple/gatsby-plugin-intl/blob/cb1148f00e3c85bafa77e2c377235019c02d552e/src/link.js#L78

That the defaultLanguage if redirect option is false don't add the language to the url? 
Thanks!

**Comment by rekuenkdr** [comment]
I agree, it would be best not to add the language for the default one to the url if redirect is set to false, so we don't have to manually make canonical links to avoid duplicity in content.

**Comment by jamie29w** [comment]
My team is experiencing the same confusion. What is the intended use of `redirect: false` if it's not this?

Btw, thank you for creating easy ergonomics in general!

**Comment by samajammin** [comment]
@rekuenkdr out of curiosity, how did you create canonical links to avoid duplicate content?

I'm looking to have all base URLs (e.g. example.com/about) set the lang path as the canonical URL (e.g. example.com/en/about).

Thanks in advance!

**Comment by samajammin** [comment]
For those looking for a solution to the duplicate content issue this creates, here's how I programmatically set the canonical URL on every page:
https://github.com/ethereum/esp-website/pull/34
